client/dataEntry.js

```diff
@@ -139,6 +139,24 @@ function getSolveWarnings(resultId, solveIndex) {
   return warnings;
 }
 
+function updateWarningContents () {
+  let data = Template.currentData();
+  let selectedResultId = selectedResultIdReact.get();
+  let warnings = getSolveWarnings(selectedResultId, data.index);
+  let $warningIcon = template.$('.solve-time-warning-icon');
```

Comment:
Hmmm... does this work? I don't see a `template` variable here, whereas there was one where you moved this from.